netdata.spec.in

```diff
@@ -10,15 +10,10 @@
 AutoReqProv: yes
 
 # This is temporary and should eventually be resolved. This bypasses
-# the default rhel __os_install_p,ost which throws a python compile
+# the default rhel __os_install_post which throws a python compile
 # error.
 %global __os_install_post %{nil}
 
-# We don’t want LTO as it has a minimal performance impact at runtime
-# but a huge impact on build times (we want our CI to not take multiple
-# hours to finish).
-%global _lto_cflags %nil
```

Comment:
So we enable lto, I can recall Costa wanted it

Reply:
It had been disabled here previously, but never got re-enabled when we decided we wanted it after all.